src/text-transform/utils.ts

```diff
@@ -1,8 +1,9 @@
+/* eslint-disable no-param-reassign */
```

Comment:
We added this rule. Why we are turning it off here?

Reply:
I have created task. We will investigate this problem and fix it